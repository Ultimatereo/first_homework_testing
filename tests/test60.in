10
151 8 -142 -213 373 878 981 -971 389 658 -743
-216 518 -655 -515 -459 -139 215 90 -595 -346 461
641 -632 -148 -445 752 248 -979 480 -651 -57 256
389 153 -764 -269 371 -216 287 585 -794 -389 992
565 458 813 -631 -73 -694 926 930 -981 -658 553
199 88 -624 -949 -880 -770 -493 231 438 759 964
689 450 295 -606 -114 -424 692 652 311 589 -549
-924 -245 -946 -500 414 -962 897 -473 164 852 -494
-787 -341 296 854 491 -586 -156 446 174 230 -697
714 -419 974 745 -99 -37 477 53 937 -492 447
