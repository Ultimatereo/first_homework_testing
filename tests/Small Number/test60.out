1.94421
-1.68362
-0.740376
2.00208
-2.37004
-0.90477
0.411817
-0.81622
-2.44552
1.11152
